src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -91,165 +106,271 @@ def _load_refcase(self, refcase_file: Optional[str]) -> Optional[EclSum]:
+        if isinstance(surface, dict):
+            name = surface.get(ConfigKeys.NAME)
+            init_file = surface.get(ConfigKeys.INIT_FILES)
+            out_file = surface.get(ConfigKeys.OUT_FILE)
+            base_surface = surface.get(ConfigKeys.BASE_SURFACE_KEY)
+            forward_int = surface.get(ConfigKeys.FORWARD_INIT)
+        else:
+            options = _option_dict(surface, 1)
+            name = surface[0]
+            init_file = options.get(ConfigKeys.INIT_FILES)
+            out_file = options.get("OUTPUT_FILE")
+            base_surface = options.get(ConfigKeys.BASE_SURFACE_KEY)
+            forward_int = _str_to_bool(options.get(ConfigKeys.FORWARD_INIT, "0"))
+
+        return EnkfConfigNode.create_surface(
+            name,
+            init_file,
+            out_file,
+            base_surface,
+            forward_int,
+        )
+
+    @staticmethod
+    def get_field_node(
+        field: Union[dict, list], grid: Optional[EclGrid], trans_table: Any
+    ) -> EnkfConfigNode:
+        if isinstance(field, dict):
+            name = field.get(ConfigKeys.NAME)
+            var_type = field.get(ConfigKeys.VAR_TYPE)
+            out_file = field.get(ConfigKeys.OUT_FILE)
+            enkf_infile = field.get(ConfigKeys.ENKF_INFILE)
+            forward_init = field.get(ConfigKeys.FORWARD_INIT)
+            init_transform = field.get(ConfigKeys.INIT_TRANSFORM)
+            output_transform = field.get(ConfigKeys.OUTPUT_TRANSFORM)
+            input_transform = field.get(ConfigKeys.INPUT_TRANSFORM)
+            min_ = field.get(ConfigKeys.MIN_KEY)
+            max_ = field.get(ConfigKeys.MAX_KEY)
+            init_files = field.get(ConfigKeys.INIT_FILES)
+        else:
+            name = field[0]
+            var_type = field[1]
+            out_file = field[2]
+            enkf_infile = None
+            options = _option_dict(field, 2)
+            if var_type == ConfigKeys.GENERAL_KEY:
+                enkf_infile = field[3]
+            forward_init = _str_to_bool(options.get(ConfigKeys.FORWARD_INIT, "0"))
+            init_transform = options.get(ConfigKeys.INIT_TRANSFORM)
+            output_transform = options.get(ConfigKeys.OUTPUT_TRANSFORM)
+            input_transform = options.get(ConfigKeys.INPUT_TRANSFORM)
+            min_ = options.get(ConfigKeys.MIN_KEY)
+            max_ = options.get(ConfigKeys.MAX_KEY)
+            init_files = options.get(ConfigKeys.INIT_FILES)
+        return EnkfConfigNode.create_field(
+            name,
+            var_type,
+            grid,
+            trans_table,
+            out_file,
+            enkf_infile,
+            forward_init,
+            init_transform,
+            output_transform,
+            input_transform,
+            min_,
+            max_,
+            init_files,
+        )
+
+    @classmethod
+    def _get_schedule_file_node(
+        cls, schedule_file: Union[dict, list, str], tag_format: str
```

Comment:
is it possible that this one can be a static method as well? doesn't look like cls is actually used. sorry didn't catch it first time around